explode/eopkg.go

```diff
@@ -27,7 +27,7 @@ import (
 // Eopkg will explode all eopkgs passed to it and return the path to
 // the "root" to walk.
 func Eopkg(pkgs []string) (string, error) {
-	rootDir, err := ioutil.TempDir("", "abireport-eopkg")
+	rootDir, err := ioutil.TempDir("/var/tmp", "abireport-rpm")
```

Comment:
Because this is in the eopkg handling code, I think it would be better to keep the second argument unmodified, or else use a generic string like `"abireport"`.

Reply:
My bad, copy & paste. I will fix this.